docs/streams/upgrade-guide.html

```diff
@@ -20,6 +20,11 @@
 <script id="content-template" type="text/x-handlebars-template">
     <h1>Upgrade Guide &amp; API Changes</h1>
 
+    <p>
+        If you want to upgrade from 1.0.x to 1.1.0 you don't need to do any code changes as the public API is fully backward compatible.
+        See <a href="#streams_api_changes_110">below</a> a complete list of 1.1.0 API and semantic changes that allow you to advance your application with the usage of new features.
```

Comment:
Missing "for" before "a complete list"? 

This sentence is a little wordy. Could we cut off "with the usage of new features"?